src/js/components/help/__snapshots__/system-updates.test.js.snap

```diff
@@ -58,7 +58,7 @@ exports[`SystemUpdates Component renders correctly 1`] = `
     You can learn more about Mender's A/B partition architecture
      
     <a
-      href="https://docs.mender.io/architecture/overview"
+      href="https://docs.mender.io/overview/introduction"
```

Comment:
This this one need a `${docsVersion}`

Reply:
the `.snap` files are the results of the test renderings - and for all the help pages the input data from [here](https://github.com/mendersoftware/gui/blob/master/src/js/components/help/mockData.js#L2) is used, which defines the docsVersion as just `''` - so the version path is omitted 👌